tests/components/conversation/test_init.py

```diff
@@ -35,7 +35,11 @@
 from tests.components.light.common import MockLight
 from tests.typing import ClientSessionGenerator, WebSocketGenerator
 
-AGENT_ID_OPTIONS = [None, conversation.HOME_ASSISTANT_AGENT]
+AGENT_ID_OPTIONS = [
```

Comment:
All these values for agent_id resolve to the default agent and all tests test are run once for each value